### DISCUSSION TITLE
Dados da conta bancária para envio do montante resultante da compra de ingressos da pyne

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#86] [Geral]
Olá,

Estamos usando a plataforma Eventbrite para venda de ingressos. Solicito os dados da conta bancária da APyB para recebimento do montante final da venda.

**Comment by cecivieira** [comment]
oi @muriloviana 
posso usar os mesmos dados bancários presente em https://apyb.python.org.br/guias/como-organizar-eventos/#quais-os-dados-da-pessoa-juridica-da-apyb ?

**Comment by muriloviana** [comment]
Oi @cecivieira, pode sim. Esses são os dados da conta da APyB 👍

**Comment by cecivieira** [comment]
Obrigada @muriloviana !!!
